packages/web-components/src/components/va-checkbox-group/va-checkbox-group.css

```diff
@@ -0,0 +1,41 @@
+@import '../../mixins/accessibility.css';
+
+:host {
+  display: block;
+  border: none;
+  padding: 0;
+  margin-top: 3rem;
+}
+
+:host([error]) {
```

Comment:
This isn't limited to the checkbox group, but I noticed that an empty string for the `error` prop gives us the error styling without the message:

![image](https://user-images.githubusercontent.com/2008881/145083085-c1bde594-f41e-4b3f-846c-2a10cfaa732d.png)


This may be something that doesn't happen in practice - wherever `error` comes from externally may ensure that the message is `null` if there isn't an actual issue. But we could be extra cautious and handle it even when it's an empty string if we want to.

Reply:
Making these changes on #236 since that's where the style updates are being made